src/components/AssetInfo/TokenSideBar.test.tsx

```diff
@@ -63,20 +63,20 @@ describe("tokenSideBar", () => {
     expect(wrapper.find("TokenSideBarHolder")).toHaveLength(1);
     expect(wrapper.find("TokenSideBarHolder section")).toHaveLength(1);
 
-    expect(wrapper.find("#sec-transferownership h4").text()).toStrictEqual("Transfer Holdership");
-    expect(wrapper.find("#sec-transferownership button").text()).toStrictEqual("Transfer");
+    expect(wrapper.find("#sec-transferholdership h4").text()).toStrictEqual("Transfer Holdership");
+    expect(wrapper.find("#sec-transferholdership button").text()).toStrictEqual("Transfer");
   });
 
-  // admin address === holder, change bene exists
+  // admin address === holder, endorse change of bene exists
   it("should show holder role, holder view, only 2 correct fields", () => {
     const wrapper = mount(
       <TokenSideBar
-        adminAddress="0xE94E4f16ad40ADc90C29Dc85b42F1213E034947C"
-        registryAddress="0xA"
+        adminAddress="0xA"
+        registryAddress=""
```

Comment:
regisrty is compulsory field. without registry there would be no rendering of the child component. So please keep it in all the fields.